app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -186,9 +190,11 @@ data class Patient(
           .map(::queryModelsToPatientProfiles)
     }
 
-    fun patientProfile(patientUuid: UUID): Flowable<PatientProfile> {
+    fun patientProfile(patientUuid: UUID): Observable<Optional<PatientProfile>> {
```

Comment:
Earlier, that method was only being used in tests and we were forcefully getting a `PatientProfile` (by using `.first()` on the returned list).

Now since we are using it in the production source set, changed it to an `Optional` since we don't know in what use cases we will be using this method.